tests/benchmark.test.ts

```diff
@@ -36,38 +35,34 @@ describe("RPCHandler", () => {
   });
 
   it("executing without an rpc timeout", async () => {
-    await jest.isolateModulesAsync(async () => {
-      const module = await import("../types/rpc-handler");
-      const rpcHandler = new module.RPCHandler({
-        ...testConfig,
-        rpcTimeout: 99999999,
-      });
-      await rpcHandler.testRpcPerformance();
-      const latencies = rpcHandler.getLatencies();
+    const module = await import("../types/rpc-handler");
+    const rpcHandler = new module.RPCHandler({
+      ...testConfig,
+      rpcTimeout: 99999999,
```

Comment:
Seems like an unusual duration.

Reply:
Yes not sure why that was there, changed it to something that makes sense.